### PR TITLE
Create resuable workflow for setting automatic_deploy_enabled

### PR DESCRIPTION
**Pull request by fredericfran-gds**

### CODE REVIEW

**Review by nsabri1 — APPROVED** [review]
OK

File: .github/workflows/set-automatic-deploys-enabled.yaml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+  workflow_call:
+    inputs:
+      resetImageTag:
+        description: 'Whether to reset image tag to main'
+        required: false
+        default: false
+        type: boolean
+      automaticDeploysEnabled:
+        description: 'Set whether automatic deploys is active or not'
+        required: false
```
Comment (sengi) [inline .github/workflows/set-automatic-deploys-enabled.yaml:54]: Would it make sense for this to be required? (given that the purpose of the job is to set it one way or the other? Or are they other usability implications of it being a required input?)

Similar question for `environment` below also.

Comment (fredericfran-gds) [reply .github/workflows/set-automatic-deploys-enabled.yaml:54]: I guess if i put required true, we don't need the default?

Comment (sengi) [reply .github/workflows/set-automatic-deploys-enabled.yaml:54]: Anyway sorry didn't mean to split hairs - I should have prefixed this with "optional"! By all means try it out either way and as long as the UI isn't super awkward it's all good with me :)

Comment (fredericfran-gds) [reply .github/workflows/set-automatic-deploys-enabled.yaml:54]: No worries, I was thinking about it before but could not see the required info. :-)

**Pull request merged**